### PR TITLE
delete tfm constants

### PR DESCRIPTION
**Pull request by kasperk81**
@ViktorHofer i found some unused constants. i think it's easier to follow with strings for the rest

### CODE REVIEW

**Review by ViktorHofer — APPROVED** [review]
Thanks a lot!

File: src/installer/tests/TestUtils/RuntimeConfig.cs [hunk 1]
```diff
@@ -279,9 +279,7 @@ public void Save()
 
             if (_tfm is not null)
             {
-                runtimeOptions.Add(
-                    Constants.Tfm.RuntimeConfigPropertyName,
-                    _tfm);
```
Comment (ViktorHofer) [inline src/installer/tests/TestUtils/RuntimeConfig.cs:284]: Are you sure that the runtime option isn't used in this test?

Comment (kasperk81) [reply src/installer/tests/TestUtils/RuntimeConfig.cs:284]: it was replaced with simple `"tfm"` on the line below

Comment (ViktorHofer) [reply src/installer/tests/TestUtils/RuntimeConfig.cs:284]: Right, sorry didn't see that.

File: src/libraries/Microsoft.Extensions.Hosting/tests/FunctionalTests/ShutdownTests.cs [hunk 2]
```diff
@@ -58,12 +58,13 @@ private async Task ExecuteShutdownTest(string testName, string shutdownMechanic)
...
 #pragma warning restore 0618
 
+            Version version = Environment.Version;
             var deploymentParameters = new DeploymentParameters(
                 applicationPath,
                 RuntimeFlavor.CoreClr,
                 RuntimeArchitecture.x64)
             {
-                TargetFramework = Tfm.NetCoreApp50,
+                TargetFramework = $"Environment{version.Major}.{version.Minor}",
```
[the remark this answers is not in the document]
Comment (kasperk81) [reply src/libraries/Microsoft.Extensions.Hosting/tests/FunctionalTests/ShutdownTests.cs:67]: ooops, https://github.com/dotnet/runtime/pull/79230. thanks for catching it

**Pull request merged**